code/modules/mob/language.dm

```diff
@@ -5,20 +5,20 @@
 */
 
 /datum/language
-	var/name = "an unknown language" // Fluff name of language if any.
-	var/desc = "A language."         // Short description for 'Check Languages'.
-	var/speech_verb = "says"         // 'says', 'hisses', 'farts'.
-	var/ask_verb = "asks"            // Used when sentence ends in a ?
-	var/exclaim_verb = "exclaims"    // Used when sentence ends in a !
-	var/whisper_verb                 // Optional. When not specified speech_verb + quietly/softly is used instead.
-	var/colour = "body"              // CSS style to use for strings in this language.
-	var/key = "x"                    // Character used to speak in language eg. :o for Unathi.
-	var/flags = 0                    // Various language flags.
-	var/native                       // If set, non-native speakers will have trouble speaking.
-	var/list/syllables               // Used when scrambling text for a non-speaker.
-	var/list/space_chance = 55       // Likelihood of getting a space in the random scramble string.
-	var/follow = 0					 // Applies to HIVEMIND languages - should a follow link be included for dead mobs?
-	var/english_names = 0			 // Do we want English names by default, no matter what?
+	var/name = "an unknown language"            // Fluff name of language if any.
+	var/desc = "A language."                    // Short description for 'Check Languages'.
+	var/speech_verb = "says"                    // 'says', 'hisses', 'farts'.
+	var/ask_verb = "asks"                       // Used when sentence ends in a ?
+	var/list/exclaim_verbs = list()             // Used when sentence ends in a !
```

Comment:
Best to put `"exclaims"` in here as a default since that's the previous behaviour as well

Reply:
Still applies here